bpf/profiling/network/netmonitor.c

```diff
@@ -273,9 +301,14 @@ static __always_inline void process_write_data(void *ctx, __u64 id, struct sock_
         return;
     }
 
-    // pid is contains
-    if (tgid_should_trace(tgid) == false) {
-        return;
+    // if connection not sent
```

Comment:
```suggestion
    // if connect event is not sent
```

I think you mean this?